openedx/core/lib/api/test_utils.py

```diff
@@ -86,6 +86,6 @@ def assertAuthDisabled(self, method, uri):
         # Django rest framework interprets basic auth headers
         # as an attempt to authenticate with the API.
         # We don't want this for views available to anonymous users.
-        basic_auth_header = "Basic " + base64.b64encode('username:password')
+        basic_auth_header = "Basic " + base64.b64encode('username:password'.encode('utf-8')).decode('utf-8')
```

Comment:
How about using `b'username:password'` to make it a bytes literal instead of encoding it after the fact?

Reply:
I think `encode` is more explicit (instead of `b` ) to indicate the type of input `b64encode` requires

Reply:
+1 for `b'username:password'`

Reply:
I'd say the same
> I think `encode` is more explicit (instead of `b` ) to indicate the type of input `b64encode` requires